src/views/account/HeaderSection.tsx

```diff
@@ -19,21 +51,60 @@ export default function HeaderSection({ item, network }: any) {
                                 {NETWORK_LIST.find((el) => el.key === network)?.name}
                             </span>
                         </div>
-                        <div className="flex items-center flex-1 gap-2 break-words">
-                            <span className="text-sm leading-5 break-all text-dark-600"><Address text={item?.address}></Address></span>
-                            <CopyButton text={item?.address} />
-                            <button className="outline-none focus:outline-none ring-0 focus:ring-0">
-                                <Link
-                                    // underline="hover"
-                                    // color="text.primary"
-                                    href={`${NETWORK_SCANNER_MAP[network]}/address/${item?.address}`}
-                                    aria-current="page"
-                                    className="text-blue-200"
-                                    target="_blank"
-                                >
-                                    <img src={getExplorerLogo(network)} style={{height: '16px', width: '16px'}} alt="" />
-                                </Link>
-                            </button>
+                        <div className='w-full flex flex-col max-md:gap-[1rem] md:flex-row justify-between'>
```

Comment:
Ideally the entire Header tag can be moved to a different component which sits in the global folder inside components.

Reply:
common component made in global folder , have commented off the duplicated header sections, file can be deleted if global one looks good - [cf8bf85](https://github.com/jiffy-labs/jiffyscan-frontend/pull/129/commits/cf8bf85bc6ecfbfc23f5300811992a8eb84a22de)